tests/algorithm/README.md

```diff
@@ -0,0 +1,99 @@
+# Record Linkage Algorithm Testing
+
+This repository contains a project to test the effectiveness of the RecordLinker algorithm.
+
+## Prerequisites
+
+Before getting started, ensure you have the following installed:
+
+- [Docker](https://docs.docker.com/engine/install/)
+- [Docker Compose](https://docs.docker.com/compose/install/)
+
+## Directory Structure
+
+- `/`: Contains the `.env` file and `Dockerfile` to build
+- `configurations/`: Contains the configuration file for the algorithm tests
+- `data/`: Contains the data `.csv` files used for the algorithm tests (seed file and test file)
+- `results/`: Contains the results of the algorithm tests
+- `scripts/`: Contains the scripts to run the algorithm tests
+
+## Steup
+
+1. Build the Docker images:
+
+    ```bash
+    docker compose --profile algo-test build
+    ```
+
+2. Configure environment variables
+
+    `/algo.env`
+    
+    Edit the environment variables in the file
+
+3. Edit the algorithm configuration file
+
+    `/configurations/algorithm_configuration.json`
+
+    Edit the configuration file to tune the algorithm parameters
+
+## Running Algorithm Tests
+
+1. Run the test
+
+    Make the script executable
+    ```bash
+    chmod +x scripts/run_test.py
+    ```
+
+    Run the test
+    ```bash
+    docker compose run --rm algo-test-runner scripts/run_test.py
+    ```
+
+2. Analyze the results
+
+    The results of the algorithm tests will be available in the `results/output.csv` file.
+
+    The results will be in a csv formatted file withthe following columns: `Test Case #`, `Expected Result`, `Match Results`, and `Error`.
```

Comment:
```suggestion
    The results will be in a CSV formatted file with the following columns: `Test Case #`, `Expected Result`, `Match Results`, and `Error`.
```

Reply:
We may want to edit these column names over time to align with naming conventions of performance metrics used for evaluating classification models, like the Record Linkage algorithm (3 classes: `match`, `no_match`, `possible_match`). E.g. `True Label` instead of `Expected Result` and `Prediction` instead of `Match Results`.